templates/publisher/market.html

```diff
@@ -11,7 +11,19 @@
         <p><a href="/account">My snaps&nbsp;&rsaquo;</a></p>
         <h1 class="u-no-margin--top">{{ title }}</h1>
       </div>
+      {% if not uploaded %}
```

Comment:
Not a big thing, but I think it would look better to have error notification below tab navigation, because it says about content of particular tab.

Same goes to same notification on measure page.

On the other hand maybe there should be no tabs or no content at all if snap is not uploaded? Maybe it should be 404? But that's probably a question to design how these pages should look for not uploaded snaps.

Reply:
Done